core/modules/layout/plugins/context/entity_layout_context_handler.inc

```diff
@@ -42,4 +39,42 @@ class EntityLayoutContext extends LayoutContext {
       return parent::label();
     }
   }
+
+  function form(&$form, &$form_state) {
+    parent::form($form, $form_state);
+
+    $form['entity'] = array(
+      '#title' => t('Enter the title or ID of a @entity entity', array('@entity' => $this->plugin)),
```

Comment:
I realize that this is the accurate way to do things, but I thought that we were avoiding the word "entity" in user-facing text. How about we drop it?

Also, the machine names of entity types are output here:

file -> 👍 
node -> use "piece of content" instead?
taxonomy_term -> use "taxonomy term"
user -> 👍

Reply:
I'm not sure about this. Any type of contexts could be added to this list. Other things could also be considered "pieces of content". Not sure.